chain/src/chain.rs

```diff
@@ -912,6 +930,55 @@ impl ChainService {
 
         debug!("}}");
     }
+
+    /// in IBD, print 1st block of the epoch and print statistics of previous epoch
+    fn update_print_statics(
+        &mut self,
+        block: &Arc<BlockView>,
+        tip_header: &HeaderView,
+        total_difficulty: U256,
+    ) {
+        let tip_epoch = tip_header.epoch();
+
+        if tip_epoch.index() == 0 {
```

Comment:
It is recommended to change the comparison logic to compare block number against epoch starting number plus epoch length, which will reduce the complexity of the code.

Reply:
Good advice!
i refactor this part,  please help to check.
i'm not sure about if the code is the best way to tell first_block/last_block number of each epoch.